cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
@karenc I do think that I am not correctly testing. However, I don't see how it is wrong to test the result by directly calling republish_collection. The bug is not whether the trigger is fired, but whether the trigger is correctly doing what it is supposed to do. Therefore, I am going to modify the test that I originally modified.